SourceCode/frontend/package.json

```diff
@@ -39,5 +39,10 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "@testing-library/react": "^10.0.2",
+    "jest-environment-jsdom-sixteen": "^1.0.3",
+    "test-data-bot": "^0.8.0"
```

Comment:
These are for testing. `@testing-library/react` and `jest-environment-jsdom-sixteen` comes bundled with create react app.  `test-data-bot` makes random mock data.